subxt/src/client/light_client/mod.rs

```diff
@@ -115,11 +163,16 @@ impl<T: Config> LightClient<T> {
     pub fn runtime_api(&self) -> RuntimeApiClient<T, Self> {
         <Self as OfflineClientT<T>>::runtime_api(self)
     }
+
+    /// Returns the chain ID of the current light-client.
+    pub fn chain_id(&self) -> subxt_lightclient::ChainId {
+        self.raw_rpc.chain_id()
```

Comment:
Does it make more sense now to save just the actual `chain_id` in `LightClient` instead of the `raw_rpc` thing now?